src/hooks/useNotifications.tsx

```diff
@@ -152,7 +154,7 @@ export function useNotifications() {
   }, [accountHeaders, data?.id, graphQLClient]);
 
   return useQuery(['notifications'], queryForNotifications, {
```

Comment:
Unrelated fix: This was attempting to fetch notifications before the app was fully logged in.